wagn/features/step_definitions/wagn_steps.rb

```diff
@@ -154,6 +154,13 @@
   sleep period.to_i
 end
 
+When /^I wait until ajax response done$/ do
+  Timeout.timeout(Capybara.default_wait_time) do
+    while page.evaluate_script('jQuery.active') != 0 do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not use `do` with multi-line `while`.